engine/common/soundlib/snd_ogg_opus.c

```diff
@@ -0,0 +1,281 @@
+/*
+snd_ogg_opus.c - loading and streaming of Ogg audio format with Opus codec
+Copyright (C) 2024 SNMetamorph
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+*/
+
+#include "soundlib.h"
+#include "crtlib.h"
+#include "ogg_filestream.h"
+#include <opusfile.h>
+#include <string.h>
+#include <stdint.h>
```

Comment:
Same about stdint.h here. 

Also, I think it's a good idea to put system headers before ours, just in case that we don't suddenly redefine something in them.